src/commands/math/advancedSymbols.js

```diff
@@ -3,7 +3,6 @@
  ***********************************/
 
 LatexCmds.notin =
-LatexCmds.sim =
```

Comment:
No, stock MathQuill includes both basic and advanced symbols, `LatexCmds.sim` just got moved to the basic symbols list